modules/apps/31-ibc-query/ibc_module.go

```diff
@@ -195,7 +195,7 @@ func (im IBCModule) OnAcknowledgementPacket(
 		return sdkerrors.Wrapf(sdkerrors.ErrUnknownRequest, "cannot unmarshal ICS-20 transfer packet data: %s", err.Error())
```

Comment:
This part is not modification of this pr, but we should modify this error message.. "ICS-20 transfer"

Reply:
In this ibc_module.go, there are a lot of comments related in transfer. We should remove them

Reply:
https://github.com/cosmos/ibc-go/blob/main/docs/ibc/apps.md#acknowledgements

Also, Acknowledgement is response of receiving packet from module.
But our ibc-query module doesn't receive acknowledgement because queried module doesn't receive packet..
So we don't need to unmarshal packet data in OnAcknowledgmentPacket function.

In other words, we do not use OnAcknowledgmentPacket fuction so just implement simple
(not unmarshal packet data)